web/src/lib.rs

```diff
@@ -354,11 +354,7 @@ impl RuffleHandle {
 
     async fn run_context_menu_callback_paste(&self, index: usize) {
         let window = web_sys::window().expect("Missing window");
-        let Some(clipboard) = window.navigator().clipboard() else {
-            tracing::warn!("Clipboard unsupported");
-            let _ = self.with_instance(|inst| inst.js_player.display_clipboard_modal(false));
```

Comment:
Is `display_clipboard_modal` still called with `false` anywhere? If not, I suppose it can be simplified by removing its parameter. Doesn't have to be in this PR though.

Reply:
You are right, I think it's unused now.

I was actually wondering if we should somehow  manually (probably something like `"clipboard" in navigator`) keep checking if `clipboard` exists.

Reply:
And I was wondering - since accessing it no longer returns an `Option`, what happens if it's not available anyway? A JS exception? Errors on use? :thinking:

Reply:
I've added an explicit check for `"clipboard" in navigator`, because otherwise the `navigator.clipboard()` just panics when the property doesn't exist.